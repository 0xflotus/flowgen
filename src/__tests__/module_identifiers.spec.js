// @flow

import { compiler, beautify } from "..";

it("should handle utility types", () => {
  const ts = `
import {ReactNode} from 'react'
import * as React from 'react'
declare function s(node: ReactNode): void;
declare function s(node: React.ReactNode): void;
`;
  const result = compiler.compileDefinitionString(ts);
  expect(beautify(result)).toMatchSnapshot();
});
