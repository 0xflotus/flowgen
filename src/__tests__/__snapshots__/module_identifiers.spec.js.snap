// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle utility types 1`] = `
"import { Node } from \\"react\\";
import * as React from \\"react\\";
declare function s(node: Node): void;
declare function s(node: React.ReactNode): void;
"
`;
